libs/commands/init/src/command.ts

```diff
@@ -32,9 +32,8 @@ const command: CommandModule = {
       type: "boolean",
     },
   },
-  handler(argv) {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    return require(".")(argv);
+  async handler(argv) {
+    (await import(".")).factory(argv);
```

Comment:
@amorscher Does this need to have a `return` before this line or is that not relevant?

Reply:
@fahslaj In this case it is not relevant it would cause a type error as init command is not inheriting from command and thus it is not a promise having functions satisfying handler type ( void | promise<void>).
 Init command is directly executed in its constructor. 

I could add an explicit return after the import. with a comment explaining the reason for it? Or maybe you have a better solution? It is a little bit inconsistent with the general approach. I could also make initcmd a promise like class.

Reply:
I updated it to make the command fulfilling promise signature so it could be returned in the handle method. So I think now it is more consitent with other commands.